alibi_detect/cd/tensorflow/spot_the_diff.py

```diff
@@ -129,7 +129,14 @@ def __init__(
             if len(initial_diffs) != n_diffs:
                 raise ValueError("Should have initial_diffs.shape[0] == n_diffs")
 
+        # Define interpretable classifier model
         model = SpotTheDiffDriftTF.InterpretableClf(kernel, x_ref_proc, initial_diffs)
+        # Build the model so that it can be passed to ClassifierDriftTF
+        model(x_ref_proc)
```

Comment:
Performance-wise, it would be preferable to do `model.build(x_ref_proc.shape)` instead of `model(x_ref_proc)`. However, `build` doesn't currently work due to the design of `GaussianRBF`. It would probably also be too restrictive on user-provided custom `kernel`'s.